internal/state/execution.go

```diff
@@ -433,14 +437,17 @@ func buildLastCommitInfo(block *types.Block, store Store, initialHeight int64) a
 
 func extendedCommitInfo(c abci.CommitInfo, votes []*types.Vote) abci.ExtendedCommitInfo {
```

Comment:
Have you considered passing only the extensions here?

Reply:
Is this still relevant, given your intended changes in your upcoming patch?

Reply:
Good point. I don't think so